environment.yml

```diff
@@ -19,6 +19,7 @@ dependencies:
   - pre-commit
   - pylint
   - flake8
+  - autoflake
```

Comment:
Was adding this here really necessary? Pre-commit should take care of it in its own virtualenv?

Reply:
Good point, sure!